src/screens/AddVocab/AddVocab.jsx

```diff
@@ -16,13 +18,12 @@ const AddVocab = () => {
   const [groups] = useState([]);
   const [groupName, setGroupName] = useState("");
   const [foreignWord, setForeignWord] = useState("");
-  const [, setTranslations] = useState([]);
+  const [translations, setTranslations] = useState([""]);
   const [description, setDescription] = useState("");
   const jwt = useSelector((state) => state.login.user.jwt);
   const serverAddress = useSelector((state) => state.login.serverAddress);
 
-  // default number of translation inputs to render
-  const translationInputs = [1, 2, 3];
+  const maxTranslations = 10;
```

Comment:
could we move this to the constants file?

Reply:
good point -> done.